concent_api/core/views.py

```diff
@@ -93,6 +99,17 @@ def receive(request, _message):
         set_message_as_delivered(last_undelivered_message_status)
         return handle_receive_force_get_task_result_upload_for_provider(request, decoded_message_data)
 
+    if isinstance(decoded_message_data, message.concents.ForceSubtaskResults):
+        set_message_as_delivered(last_undelivered_message_status)
+        if current_time < decoded_message_data.timestamp + settings.CONCENT_MESSAGING_TIME:
```

Comment:
According to https://github.com/golemfactory/concent/issues/44 we always want to return the message if it is queued.